core/templates/dev/head/pages/header_css_libs.html

```diff
@@ -1,11 +1,8 @@
 <link rel="stylesheet" type="text/css" media="screen"
       href="https://fonts.googleapis.com/css?family=Capriola|Rubik|Roboto|Material+Icons|Open+Sans:400,600">
-{% if not DEV_MODE %}
-  <link rel="stylesheet" type="text/css" media="screen"
+<link ng-if="!DEV_MODE" rel="stylesheet" type="text/css" media="screen"
       href="{{get_static_resource_url('/third_party/generated/css/third_party.min.css')}}">
-{% else %}
-  <link rel="stylesheet" type="text/css" media="screen"
+<link ng-if="DEV_MODE" rel="stylesheet" type="text/css" media="screen"
```

Comment:
The same problem is here *header_css_libs.html* is included earlier in base.html https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/base.html#L72 than https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/base.html#L118